apps/system/js/system_banner.js

```diff
@@ -28,29 +34,36 @@
     _clicked: false,
 
     /**
-     * Callback when the user clicks on the system banner button.
+     * Set to true when the banner is currently showing to the user.
      * @memberof SystemBanner.prototype
-     * @type {Function}
+     * @type {Boolean}
      */
-    _clickCallback: null,
+    _showing: false,
 
     /**
      * Generates and returns the banner if it does not exist
      * @memberof SystemBanner.prototype
      * @type {Function}
      */
-    get banner() {
-      if (!this._banner) {
-        this._banner = document.createElement('section');
-        this._banner.className = 'banner generic-dialog';
-        this._banner.setAttribute('role', 'dialog');
-        this._banner.dataset.zIndexLevel = 'system-notification-banner';
-        this._banner.dataset.button = 'false';
-        this._banner.innerHTML = '<p></p><button></button>';
-        document.getElementById('screen').appendChild(this._banner);
+    getBanner: function() {
+      if (this._banner) {
+        return Promise.resolve(this._banner);
       }
 
-      return this._banner;
+      return LazyLoader.load('shared/elements/gaia-component/gaia-component.js')
```

Comment:
Unfortunately no, gaia-toast depends on gaia-component, and I found when I loaded them both in an array I would sometimes get an error related to `require('gaia-component')` being undefined.